packages/core/template_project/app/build.gradle

```diff
@@ -46,7 +47,7 @@ android {
     compileSdkVersion 29
     defaultConfig {
         applicationId "<%= packageId %>"
-        minSdkVersion 19
+        minSdkVersion 21
```

Comment:
Is `minSdkVersion 21` a requirement for location delegation? If that's the case and it's not possible to maintain it, I'd only use 21 when notification delegation is enabled and user 19 otherwise. Sounds worth warning developers that enabling notification delegation will make the app incompatible with some devices too. According to Android Studio, we'd be dropping from supporting 98.1% of devices to 94.1%.

![Screenshot 2020-07-30 at 13 52 41](https://user-images.githubusercontent.com/1733592/88925095-fd3a2000-d26b-11ea-9cb1-217ab8a343b9.png)